_maps/shuttles/construction_mining.dmm

```diff
@@ -0,0 +1,145 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 9
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"b" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"c" = (
+/obj/machinery/computer/shuttle{
+	possible_destinations = "construction_mining_custom";
+	shuttleId = "construction_mining"
+	},
```

Comment:
I'm not allowing player-built shuttles with custom docking locations. This `construction_mining_custom` looks suspiciously like a player-defined docking location.

You're advertising this as a buyable spare mining shuttle, so it should be limited to all the same docking ports as the existing mining shuttle.

If this means you need to add a secondary docking areas to allow this to function properly, then that's fine.

Also this should not be a var edit hidden in a mapping file.